edk2toolext/environment/uefi_build.py

```diff
@@ -172,10 +172,20 @@ def Go(self, WorkSpace, PackagesPath, PInHelper, PInManager):
                     logging.critical("Flash Image failed")
                     return ret
 
-        except:
+        except Exception:
             logging.critical("Build Process Exception")
             logging.error(traceback.format_exc())
             return -1
+
+        except SystemExit:
```

Comment:
These were suggested additions? Or you decided to expand the checks since you were already hitting this file?

Reply:
as part of changing from handling all exceptions to handling the three base types (which was the suggestion) I added unique messages so it could be determined what happened.  So it isn't really much of an expansion.